# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/RotatedBuffer.cpp
# Commit: 75b04de7e99c
# Full Hash: 75b04de7e99cab7bed5cc8b0f2f6d52ddf174bf3
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-25 09:39:13
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r 91c3acdb2454 -r 75b04de7e99c gfx/layers/RotatedBuffer.cpp
--- a/gfx/layers/RotatedBuffer.cpp	Fri May 24 11:32:20 2019 +0000
+++ b/gfx/layers/RotatedBuffer.cpp	Fri May 24 11:32:54 2019 +0000
@@ -12,7 +12,6 @@
 #include "GeckoProfiler.h"          // for AUTO_PROFILER_LABEL
 #include "Layers.h"                 // for PaintedLayer, Layer, etc
 #include "gfxPlatform.h"            // for gfxPlatform
-#include "gfxPrefs.h"               // for gfxPrefs
 #include "gfxUtils.h"               // for gfxUtils
 #include "mozilla/ArrayUtils.h"     // for ArrayLength
 #include "mozilla/gfx/BasePoint.h"  // for BasePoint
@@ -25,6 +24,7 @@
 #include "mozilla/layers/ShadowLayers.h"   // for ShadowableLayer
 #include "mozilla/layers/TextureClient.h"  // for TextureClient
 #include "mozilla/Move.h"                  // for Move
+#include "mozilla/StaticPrefs.h"           // for StaticPrefs
 #include "mozilla/gfx/Point.h"             // for IntSize
 #include "gfx2DGlue.h"
 #include "nsLayoutUtils.h"  // for invalidation debugging
@@ -76,7 +76,7 @@
   MOZ_ASSERT(!mCapture);
   MOZ_ASSERT(target);
   mCapture = Factory::CreateCaptureDrawTargetForTarget(
-      target, gfxPrefs::LayersOMTPCaptureLimit());
+      target, StaticPrefs::LayersOMTPCaptureLimit());
 }
 
 RefPtr<gfx::DrawTargetCapture> RotatedBuffer::EndCapture() {